src/xcode/ENA/ENA/Source/Services/Risk/CalculationV2/RiskCalculationWindow.swift

```diff
@@ -0,0 +1,125 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+
+/// Determines the risk level for one exposure window
+/// https://github.com/corona-warn-app/cwa-app-tech-spec/blob/7779cabcff42afb437f743f1d9e35592ef989c52/docs/spec/exposure-windows.md#determine-risk-level-for-exposure-windows
+final class RiskCalculationWindow {
```

Comment:
I'm not overly happy with the name of this. I would suggest to avoid using the term `Window` outside of `ExposureWindow` unless really necessary. How about `ExposureWindowRiskAssessment`? Or `ExposureWindowRiskWrapper`?

Reply:
In the caller context the window naming feels the most logical to us. If we rename this, we would have `filteredExposureWindowRiskAssessments` and `exposureWindowsRiskAssessmentsPerDate` or something similar.

Reply:
Well, but then you should also have e.g.`filteredRiskCalculationWindows`. Would `ExposureWindowRiskWrapper` or `ExposureWindowCalculationWrapper` to so bad? If it's "just a wrapper", I think calling variables `filteredExposureWindows` is still fine.

Reply:
We could go the other direction and make it an exposure window using the decorator pattern. We gave it a try and called it `RiskCalculationExposureWindow` and think this accurately describes the exposure window that is extended with risk calculation features. What do you think about this approach?

Reply:
Fine for me 👍